.github/workflows/healm_publish.yaml

```diff
@@ -0,0 +1,29 @@
+name: helm-publish
```

Comment:
to be able to reference direct link to helm chart from release assets: https://github.com/ddelange/docker-registry.helm/releases/tag/v2.2.3

since we can release from any branch, this allowed me to test this PR on our live staging cluster